wrapper/src/test/java/integration/refactored/container/tests/ReadWriteSplittingTests.java

```diff
@@ -166,6 +185,76 @@ protected String getProxiedUrl() {
         + DriverHelper.getDriverRequiredParameters();
   }
 
+  @TestTemplate
+  @EnableOnDatabaseEngine(DatabaseEngine.MYSQL)
+  // Tests use Aurora specific SQL to identify instance name
+  @EnableOnDatabaseEngineDeployment(DatabaseEngineDeployment.AURORA)
+  public void test_connectToWriter_setReadOnlyTrueFalseTrue(final Properties props) throws SQLException {
```

Comment:
A better test name?

Reply:
Is it the same test as test_connectToWriter_setReadOnlyTrueTrueFalseFalseTrue()?

Reply:
test_connectToWriter_setReadOnlyTrueTrueFalseFalseTrue does everything that test_connectToWriter_setReadOnlyTrueFalseTrue does, since it checks switching from true-> false, false->true, true->true, false-> false 
Therefore I will delete test_connectToWriter_setReadOnlyTrueFalseTrue